framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -59,7 +59,31 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
      * @type {Integer} Factor which is used for adapting the delta of the mouse wheel
      * event to the roll events,
      */
-    ROLL_FACTOR : 18
+    WHEEL_ROLL_FACTOR: 18,
```

Comment:
I agree. Although the new name is more descriptive, which I almost always prefer, in this case, backward-compatibility should trump that.